### PR TITLE
functionalities of the final part of the app

### PR DESCRIPTION
**Pull request by Matarratas777**
Navigation and different functionalities were implemented in the app, final functionalities in it, as well as properties for the use of components were created.

In this section a visual section is not involved but more code

### CODE REVIEW

**Review by Manz1999 — APPROVED** [review]

**Review by Zuack55 — APPROVED** [review]
niceee

**Review by McFly1208 — APPROVED** [review]
LGTM

File: App.js [hunk 1]
```diff
@@ -8,11 +8,14 @@ import PantallaOrigen from './src/views/pantallaOriDes/pantallaOrigen'
...
 export default function App() {
   return (
-  
-      <Navigation/>
-    
+    // <PantallaOrigen />
+    // <PantallaOriDes />
+    // <PantallaCalendario />
+    <Navigation />
+    // <Vuelos />
```
Comment (IvanGodinez21) [inline App.js:19]: You should remove comments

**Review by arthurchina92 — APPROVED** [review]
LGTM

**Review by IvanGodinez21 — APPROVED** [review]
Just check my comments, LGTM! 🐒

**Review by RobertoCarMed — APPROVED** [review]

File: App.js (hunk 1 above)
Comment (Matarratas777) [reply App.js:19]: okey
🍆

**Pull request merged**